deltacat/compute/compactor/compaction_session.py

```diff
@@ -497,12 +622,22 @@ def _execute_compaction_round(
         if round_completion_info
         else None
     )
+
+    pyarrow_write_result = PyArrowWriteResult.union(
+        [m.pyarrow_write_result for m in mat_results]
+    )
+
+    compaction_audit.set_output_file_count(pyarrow_write_result.files)
+    compaction_audit.set_output_size_bytes(pyarrow_write_result.file_bytes)
+    compaction_audit.set_output_size_pyarrow_bytes(pyarrow_write_result.pyarrow_bytes)
+
```

Comment:
Maybe also roll this into a single method like `compaction_audit.save_write_result(pyarrow_write_result)`.

Reply:
Done. We can create somewhat generic methods like this. I created individual setters to not couple to the compaction session implementation.